src/view/ThemeManager.js

```diff
@@ -237,6 +237,7 @@ define(function (require, exports, module) {
                     return result.content;
                 })
                 .then(function (cssContent) {
+                    $("body").toggleClass("dark", theme.dark);
```

Comment:
this fix seems to assume a binary theme choice in that if there is any theme at all, it must be the dark theme. That will break the moment we introduce a third theme, or custom theming, so it would be better to check which theme should actually be active. Maybe @flukeout can help point in the right direction here